renormalizer/model/mlist.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, mol_list: List[Mol], j_matrix: Union[Quantity, np.ndarray, No
             for imol, mol in enumerate(mol_list):
                 if imol == len(mol_list) // 2:
                     self._e_idx = [len(self.pbond_list)] * len(mol_list)
-                    self.pbond_list.append(len(mol_list))
+                    self.pbond_list.append(len(mol_list)+1)
```

Comment:
What is this +1?

Reply:
The physical space now contains zero exciton space, the basis are: |vacuum>, |1>...|m>